{
  "Header": {
    "Original": "CVE-2016-8613 (A flaw was found in foreman 1.5.1. The remote execution plugin runs co ...)",
    "Line": 428393,
    "ID": "CVE-2016-8613",
    "Description": "(A flaw was found in foreman 1.5.1. The remote execution plugin runs co ...)"
  },
  "Annotations": [
    {
      "Original": "- foreman \u003citp\u003e (bug #663101)",
      "Line": 428394,
      "Type": "package",
      "Package": "foreman",
      "Kind": "itp",
      "BugNo": 663101
    },
    {
      "Original": "NOTE: http://projects.theforeman.org/issues/17066/",
      "Line": 428395,
      "Type": "NOTE",
      "Description": "http://projects.theforeman.org/issues/17066/"
    },
    {
      "Original": "NOTE: https://github.com/theforeman/foreman_remote_execution/pull/208",
      "Line": 428396,
      "Type": "NOTE",
      "Description": "https://github.com/theforeman/foreman_remote_execution/pull/208"
    }
  ]
}